packages/design-system-scripts/gulp/docs/generatePages/generatePages.js

```diff
@@ -205,14 +207,28 @@ module.exports = async function generatePages(sourceDir, docsDir, options, chang
     )
   );
 
+  // 1. Are we only showing CMSDS link for patterns and components?
+
+  /* 2. If you create a duplicate page like Button at the child design system level and edit the first section of the documentation page. 
+The inheritance of the USWDS and CMSDS link breaks becasuse the child version of the page takes precendence. 
+*/
+
+  // Get pages that come from core CMS Design System
+  const corePages = markdownSections.concat(kssSections).map((page) => {
```

Comment:
`corePages` isnt really a great name for this variable, as it contains all the pages, just with some modifications for certain pages. I would rename it, or consider grouping it in with the `const pages` line below

Reply:
This is what I would suggest, it involves changing the names of 2 other variables from `pages` to `uniquePageSections` and from `nestedPages` to `pages`
```
const pageSections = markdownSections.concat(kssSections)
   .map(section => addCmsdsLink(section))

const uniquePageSections = uniquePages(pageSections).filter((page) => !page.hideSection);

const pages = await addTopLevelPages(pages).then(nestSections);
```

Reply:
On second thought maybe these names are better:
`allPageSections`
`uniquePageSections`
`nestedPageSections`